tests/client-middleware-cache.js

```diff
@@ -117,37 +123,4 @@ describe("Client Middleware", () => {
       });
     });
   });
-
```

Comment:
@michaloo the test are failing for me. Can you please have a look at it ?
Also I have created a HullStub class on the master branch to avoid repeating all this stub code on all the test files